test_sets/smoke_tests.txt

```diff
@@ -44,6 +44,7 @@
 ./test_scripts/Smoke/API/044_AlertManeuver_Non_Media_PositiveCase_SUCCESS.lua
 ./test_scripts/Smoke/API/045_PerformAudioPassThru_Non_Media_PositiveCase_SUCCESS.lua
 ./test_scripts/Smoke/API/048_GetSystemCapabilityRequest_SUCCESS.lua
+./test_scripts/API/CloseApplication/001_Success_flow.lua
```

Comment:
@jacobkeeler as I know we have no practice to add feature tests to smoke tests list. But I like this practice. Can we add other delivered features success flow to smoke_tests.txt

Reply:
Agreed, at the very least the base success cases should be added for new RPCs. It seems that this hasn't been done for the app service RPCs, so I'll submit a PR for that.